client/src/components/layout/DogShowPage.js

```diff
@@ -14,24 +23,43 @@ const DogShowPage = (props) => {
       }
       const body = await response.json();
       setDog(body.dog);
+
+      return true;
     } catch (error) {
       console.log(error);
+      return false;
     }
   };
+
   useEffect(() => {
     getDog();
   }, []);
+
+  const reviewsList = dog.reviews.map((review) => {
+    return (
+      <ReviewTile
+        key={review.id}
+        userName={review.userName}
+        description={review.description}
+        rating={review.rating}
+      />
+    );
+  });
+
+  let dogDescription = "";
+  if (dog.description) {
+    dogDescription = <p>{dog.description}</p>;
```

Comment:
Instead of not showing anything if the description is empty, you could display conditional text "no description provided", that way it doesn't look like the page failed to load or is missing data

Reply:
This would make more sense if you see the mockups. (And also our naming convention is bad.) The description is optional. If it's left empty, the name field appears as just the dog's name as a headline. With a description, there's a paragraph tag underneath.
![Screen Shot 2022-02-03 at 5 18 30 PM](https://user-images.githubusercontent.com/96082944/152438886-48a99e4a-b9af-413b-a898-daf312238829.png)
![Screen Shot 2022-02-03 at 5 19 21 PM](https://user-images.githubusercontent.com/96082944/152438892-bd809540-dd90-494a-9a93-3542e08bcfc4.png)

Reply:
Added default -"No description provided" text